tests/other_test.go

```diff
@@ -0,0 +1,78 @@
+// Tests in this file are NOT run in the PR pipeline. They are run in the continuous testing pipeline along with the ones in pr_test.go
+package test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/files"
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
+)
+
+const defaultExampleTerraformDir = "examples/basic"
+
+func sshPublicKey(t *testing.T) string {
+	prefix := fmt.Sprintf("slz-test-%s", strings.ToLower(random.UniqueId()))
+	actualTerraformDir := "./resources"
+	tempTerraformDir, _ := files.CopyTerraformFolderToTemp(actualTerraformDir, prefix)
+	logger.Log(t, "Tempdir: ", tempTerraformDir)
+
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: tempTerraformDir,
+		// Set Upgrade to true to ensure latest version of providers and modules are used by terratest.
+		// This is the same as setting the -upgrade=true flag with terraform.
+		Upgrade: true,
+	})
+
+	terraform.WorkspaceSelectOrNew(t, terraformOptions, prefix)
+	_, existErr := terraform.InitAndApplyE(t, terraformOptions)
+	if existErr != nil {
+		assert.True(t, existErr == nil, "Init and Apply of temp existing resource failed")
+	}
+
+	return terraform.Output(t, terraformOptions, "ssh_public_key")
+}
+
+func setupOptions(t *testing.T, prefix string) *testhelper.TestOptions {
+
+	sshPublicKey := sshPublicKey(t)
+
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:      t,
+		TerraformDir: defaultExampleTerraformDir,
+		Prefix:       prefix,
+		TerraformVars: map[string]interface{}{
+			"ssh_key": sshPublicKey,
+		},
+	})
+
+	return options
+}
+
+func TestRunBasicExample(t *testing.T) {
+	t.Parallel()
+
+	options := setupOptions(t, "land-zone")
+
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
+
+func TestRunUpgradeBasicExample(t *testing.T) {
```

Comment:
We don't add upgrade tests to other_test.go since they don't run against PR branches, so upgrade is not applicable (tests run against main branch only)